### PR TITLE
Re-enable unit tests for VDI.get_nbd_info

### PR DESCRIPTION
**Pull request by gaborigloi**
I've successfully compiled this and the tests passed. The Travis build isn't expected to pass, because we've been having build issues recently.

Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>

### CODE REVIEW

**Comment by gaborigloi** [comment]
It turns out that the documentation for formatting is really good: https://caml.inria.fr/pub/docs/manual-ocaml/libref/Format.html

**Comment by gaborigloi** [comment]
@mseri It turns out one can use opam as a replacement of planex-buildenv, I've used that method to build this: I've made a new switch for CBT, and pinned the `feature/CBT` branches of the repositories that we had to modify for CBT. This works quite well, and I even managed to successfully `opam upgrade` after changing the storage interface in `xcp-idl`.

File: ocaml/xapi/test_vdi_cbt.ml [hunk 1]
```diff
@@ -238,9 +246,8 @@ let test_get_nbd_info =
 
   let open OUnit in
   "test_get_nbd_info" >:::
-  [
-  (*"test_returns_correct_uris" >:: test_returns_correct_uris *)
-    "test_returns_empty_list_when_no_host_is_connected_to_sr" >:: test_returns_empty_list_when_no_host_is_connected_to_sr
+  [ "test_returns_correct_uris" >:: test_returns_correct_uris
```
Comment (thomassa) [inline ocaml/xapi/test_vdi_cbt.ml]: The element type in the list is longer a uri, so perhaps rename this?

**Comment by thomassa** [comment]
Lovely!
How do the pretty strings come out? Example?

File: ocaml/xapi/test_vdi_cbt.ml (hunk 1 above)
Comment (gaborigloi) [reply ocaml/xapi/test_vdi_cbt.ml]: Good catch, forgot that one!

**Pull request merged**